linode/manager
# upcoming: [M3-7580] - Add list view for Linode Clone/Backup

## Description 📝
Introduces a list view for selecting linodes to clone/restore from in the create UI.

This is the final PR of the [Linode Clone UI Updates](https://github.com/linode/manager/issues?q=label%3A%22Linode+Clone+UI+Updates%22).

## Changes  🔄
- Introduces a new list view for large-screen (`mdUp`) devices
- List view displays Linode label, status, image, plan and region
- Includes inline "Power Off" action button for running linodes.

## Preview 📷

| Before  | After   |
| ------- | ------- |
| Clone Linode |  |
|  ![Screenshot 2024-02-13 at 4 28 38 PM](https://github.com/linode/manager/assets/122488130/04c280d1-c289-407a-b05a-da79e8777d93) |  ![Screenshot 2024-02-13 at 4 28 02 PM](https://github.com/linode/manager/assets/122488130/c6a82081-c5fc-4f22-8456-5dc81ebee4be)
| Backups |  |
|![Screenshot 2024-02-13 at 4 28 47 PM](https://github.com/linode/manager/assets/122488130/15f56335-7b24-4c42-9e91-0a74e31df4b6)|![Screenshot 2024-02-13 at 4 28 23 PM](https://github.com/linode/manager/assets/122488130/6d3b64e7-227d-4b4c-9b17-cd1ad79321d4)|

## How to test 🧪

### Prerequisites
- Ensure "Linode Clone UI Changes" feature flag is turned on

### Verification steps 
- Enter the Linode Create flow, either through the "Create" menu on the top-left corner of the app or by selecting "Clone" in the action menu for a linode
- In both the "Clone Linode" and "Backups" tabs:
    - Linode label, status, image, plan and region are displayed
    - Sorting works as expected for all fields
    - For running Linodes: Power Off button is displayed and clicking it displays a confirmation dialog
    - For small displays, list reverts back to the original cards view
- In the "Clone Linode" tab only:
    - Selecting a linode populates the region, plan selection and label fields
- In the "Backups" tab only:
    - Selecting a linode displays all available backups for that linode
## As an Author I have considered 🤔

*Check all that apply*
